exporter/trace/zipkin/codes.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkin
+
+type code int
+
+// Canonical status code enum.
+const (
+	OK code = iota
```

Comment:
+1

Reply:
Looks like C/C++ constant formatting this way, exporting being a side effect of the formatting choice.

I'd prefer them more Go idiomatic `failedPrecondition` instead of `FAILED_PRECONDITION`

Reply:
This was so the string function could be autogenerated by the stringer tool.  Changed to use an array.